cves/CVE-2011-3192.yml

```diff
@@ -208,4 +224,8 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    The main mistake here seems to be a result of both coding errors and design
+    problems. The ability to specify overlapping byteranges in a header allows for countless
+    exhaustion issues and in lacking proper validation and restriction of inputs
```

Comment:
**Grammar**
exhaustion issues and`, in lacking proper validation and input restrictions,` ...

Reply:
Definitely an issue, thanks for pointing it out! I took it in a slightly different direction to make it more clear.